docs/how-to/namespace-transition.md

```diff
@@ -0,0 +1,286 @@
+# Transition a project to the new `events-system` namespace
+
+The upcoming `v0.20` release of `knative-gcp` will contain changes that are not
+compatible with old releases. Namely, the namespace will change from
+`cloud-run-events` to `events-system`. Using the new release without running the
+migration steps described below will result in downtime and errors produced by
+both the previous and current installation.
+
+To avoid these issue we recommend taking the proactive approach described in
+[Recommended Migration](#recommended-migration) to ensure continued good
+functionality of your installed events system.
+
+**Note that the current instructions only cover installation based on the
+release from the `knative-gcp` repository. We will update the page with specific
+`gcloud` instructions as the release becomes available there as well.**
+
+## Recommended Migration
+
+1. Run the following commands before installing the new `v0.20` release. We
+   explain the purpose of each command in comments.
+
+   ```shell script
+   # create the new namespace by copying the information from the old one
```

Comment:
Can we be more specific than "information"? Is the purpose of this line to copy labels and annotations?

Reply:
Yes. I updated the comment.